### ISSUE TITLE
issue description images from jira

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/parabol#5098]
to access jira images, you need an `Authorization` header.
that header is not easy to add for `<img>` tags. basically, we'd have to augment the service worker to intercept the image request & add the auth token. but, it's possible not all parabol users are jira users, which means we'd either have to share a jira auth token across users (not safe) or not show the image for those parabol users who aren't integrated with jira (not cool).

to avoid sharing a jira auth token, we could proxy the request through our server. 
client -> parabol. parabol looks up user, looks up issue, grabs auth token for issue -> jira -> parabol -> client.
we're still left with handling headers though since the request will come in via http & without a cookie. we could sniff for this request in the service worker & attach the auth token from local storage, but that feels pretty hacky.

next, we could give the client our own token that we can turn into a jira auth token. e.g. `action.parabol.co/jiraImage/123.png?token=foo`. our server trades in `token=foo` for the issue creator's auth token (via redis?), fetches image 123 from jira & returns it. 

we could skip the middle man & instead have that token return the image itself. redis is pretty good at storing images.

both strategies above mean the client requests an image that can be cached, which is great.
it does mean 2 round trips (1 returns the description, then the client requests all images referenced in the description).
however, we can mitigate this by returning the description with URLs before we perform the fetch from jira. it's possible that a user requests an image that we haven't gotten from jira yet, but we can store a dummy value in there that says it's on its way.
 
finally, we could just base64 inline all images in the descriptionHTML. Size would be 37% larger, which is a little painful, & can't be cached. it also means the response takes longer because we'll have to fetch all images for all issues before we can respond.

**Comment by mattkrick** [comment]
closing in favor of #5100
